extensions/blocks/calendly/index.js

```diff
@@ -57,4 +62,11 @@ export const settings = {
 			},
 		],
 	},
+	deprecated: [
+		{
+			attributes,
+			supports,
```

Comment:
Is there a reason why we are deprecating the supports?
As far as I can see, they don't change, and aren't changing, so shouldn't really be migrated. 
I've tried migrating Calendly without defining `deprecated.supports` and it seems to work fine, but maybe I'm overlooking something. 🤔

Reply:
You're right, it's not needed. Originally I tried only deprecating `save` but it also needed `attributes` so then I assumed it needed `supports` too, but I was wrong. I have pushed a commit to remove this from both blocks.